tests/utils.py

```diff
@@ -902,22 +902,38 @@ async def resource_action_consistency_check():
 
     async def get_data(postgresql_client):
         post_ra_one = await postgresql_client.fetch(
```

Comment:
Are these queries still use, I wonder?  (see method description)

Reply:
It is still being used. Although I don't know why we decided to include `resource_version_ids` in the database